volatility3/framework/symbols/windows/pdbutil.py

```diff
@@ -131,8 +131,14 @@ def get_guid_from_mz(cls, context: interfaces.context.ContextInterface, layer_na
         # Check it is actually the MZ header
         if mz_sig != b"MZ":
             return None
-
-        nt_header_start = ord(layer.read(offset + 0x3C, 1))
+        
+        nt_header_start = struct.unpack("<I", layer.read(offset + 0x3C, 4))[0]
```

Comment:
I'd be a little happier using `int.from_bytes(..., byteorder='little')`, since it doesn't require the use of an additional module or splitting a tuple, but it's not a showstopper...

Reply:
I prefer struct because it is more specific.